tests/test_broker.py

```diff
@@ -0,0 +1,86 @@
+import asyncio
+import uuid
+
+import pytest
+from taskiq import AsyncBroker, BrokerMessage
+
+from taskiq_redis.redis_broker import ListQueueBroker, PubSubBroker
+
+
+async def get_message(broker: AsyncBroker) -> BrokerMessage:
+    """
+    Get a message from the broker.
+
+    :param broker: async message broker.
+    :return: first message from listen method.
+    """
+    return await anext(broker.listen())
```

Comment:
This won't pass in python 3.7.

Reply:
To make it possible you can convert it to

```python
async for message in broker.listen():
    return message
```

Reply:
Fixed it

Reply:
Nice!